examples/quickstart/jupyter-notebooks/sql-tutorial.ipynb

```diff
@@ -0,0 +1,765 @@
+   "execution_count": null,
+   "id": "228ae0e4-355e-4b4d-8253-fb2e46715559",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "endpoint = \"/druid/v2/sql\"\n",
+    "print(f\"\\033[1mQuery endpoint\\033[0m: {druid_host+endpoint}\")\n",
+    "http_method = \"POST\"\n",
+    "\n",
+    "payload = json.dumps({\n",
+    "  \"query\":\"SELECT channel, COUNT(*) AS counts FROM \\\"wikipedia-sql-tutorial\\\" GROUP BY channel LIMIT 7\"\n",
+    "})\n",
+    "headers = {'Content-Type': 'application/json'}\n",
+    "\n",
+    "response = requests.request(http_method, druid_host+endpoint, headers=headers, data=payload)\n",
+    "\n",
+    "print(\"\\033[1mQuery\\033[0m:\\n\" + payload)\n",
+    "print(\"\\n\\033[1mResponse\\033[0m: \\n\" + json.dumps(response.json(), indent=4))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eab67db4-a0f3-4177-b5be-1fef355bf33f",
+   "metadata": {},
+   "source": [
+    "You can further define the groups by specifying multiple dimensions.\n",
+    "Druid SQL supports using numbers in GROUP BY and ORDER BY clauses to refer to column positions in the SELECT clause.\n",
+    "Similar to SQL, Druid SQL uses one-based indexing to reference elements in SQL statements.\n",
+    "\n",
+    "For example, the next cell aggregates entries grouped by fields `cityName` and `countryName`.\n",
+    "The output has three fields: `cityName`, `countryName`, and `counts`. For each unique combination of `cityName` and `countryName`, Druid aggregates all rows and averages the entries in the group.\n",
+    "The output is limited to seven rows for display purposes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ca4b3bae-2b02-4c90-98a5-cb7806b6e649",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "endpoint = \"/druid/v2/sql\"\n",
+    "print(f\"\\033[1mQuery endpoint\\033[0m: {druid_host+endpoint}\")\n",
+    "http_method = \"POST\"\n",
+    "\n",
+    "payload = json.dumps({\n",
+    "  \"query\":\"SELECT cityName, countryName, COUNT(*) AS counts FROM \\\"wikipedia-sql-tutorial\\\" GROUP BY 1, 2 LIMIT 7\"\n",
+    "})\n",
+    "headers = {'Content-Type': 'application/json'}\n",
+    "\n",
+    "response = requests.request(http_method, druid_host+endpoint, headers=headers, data=payload)\n",
+    "\n",
+    "print(\"\\033[1mQuery\\033[0m:\\n\" + payload)\n",
+    "print(\"\\n\\033[1mResponse\\033[0m: \\n\" + json.dumps(response.json(), indent=4))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9c2e56af-3fdf-40f1-869b-822ac8aafbc8",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Query types\n",
+    "\n",
+    "Druid uses the [Apache Calcite](https://calcite.apache.org/) data management framework to translate SQL statements into Druid’s [native JSON-based queries](https://druid.apache.org/docs/latest/querying/querying.html) on the Broker. The queries are then executed by the Druid cluster.\n",
```

Comment:
Why do I need to know about Calcite? How does it affect my behavior as a developer? Why does the Broker matter to me in this moment. I'd consider a revision to this to section to maybe say that Druid is optimized for certain query types (Scan, etc). Then you can say that Druid translates the SQL queries to native (using Calcite). If folks need to know about the broker, we can mention that. @2bethere  WDYT?